sql/branch.sql

```diff
@@ -0,0 +1,16 @@
+BEGIN;
+    CREATE TYPE email_auth_intent AS ENUM ('sign-in');
+
+    CREATE TABLE email_auth_nonces
+    ( id                    serial                      PRIMARY KEY
+    , email_address         text                        NOT NULL
```

Comment:
We won't always have a reference to the `emails` table, this is only for `sign-in`. (check the `sign_in_has_valid_email` constraint below).